From 6d51fe44935df9046a377bdc06d500a52f2a8a2a Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Fri, 25 Aug 2023 12:18:19 +0530
Subject: [PATCH] arm64: dts: qcom: ipq5332: Add build support for RDP479 &
 RDP481

RDP479 IPQ5332 + QCN6432 + QCN9224 (2+2+4)
	PCIe0 QCN6432-6G
	PCIe1 QCN9224-5G

RDP481 IPQ5332 + QCN6432 + QCN9224 (2+2+4)
	PCIe0 QCN6432-5G
	PCIe1 QCN9224-6G

Change-Id: I7da376612b0828ecf1c055840648f3ecba2da900
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 2 ++
 arch/arm64/boot/dts/qcom/Makefile | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 614b3c3510ee..91595ca7919d 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1080,6 +1080,8 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq5332-rdp474.dtb \
 	ipq5332-rdp477.dtb \
 	ipq5332-rdp478.dtb \
+	ipq5332-rdp479.dtb \
+	ipq5332-rdp481.dtb \
 	ipq5332-db-mi01.1.dtb \
 	ipq5332-db-mi02.1.dtb \
 	ipq5332-db-mi03.1.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 82773f6e2e58..69c97b316f67 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -12,6 +12,8 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp472.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp474.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp477.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp478.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp479.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp481.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi01.1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi02.1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi03.1.dtb
-- 
2.34.1

